client/src/pages/Home/Footer/Footer.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
```

Comment:
do we need this file at all?

Reply:
No, we don't need the following file.
 <Footer folde>
Footer.js
Footer.css  
I need to delete the folder.